modules/mod_rentals.rb

```diff
@@ -21,12 +21,12 @@ def create_rental(books, people, rentals)
     rental_storage = fetch_storage('rentals')
     puts 'Select a book to rent'
     books.each_with_index { |book, index| puts "#{index}) Book Title: #{book.title}, Author: #{book.author}" }
-    rental_book = gets.chomp
+    rental_book = gets.chomp.to_i
     puts 'Select Person by their ID'
     people.each_with_index do |person, index|
       puts "#{index}) #{[person.type]} Name: #{person.name} Age: #{person.age} Id: #{person.id}"
     end
-    rental_person = gets.chomp
+    rental_person = gets.chomp.to_i
     puts 'Enter date of Rental'
     date = gets.chomp
     rental_item = Rental.new(people[rental_person], books[rental_book], date)
```

Comment:
- [ ] As stated by the previous reviewer kindly make sure when creating a rental doesn't give an error when trying to add a date as stated in the following screen.
>![image](https://user-images.githubusercontent.com/53402600/200545057-fdc7a3d8-0df5-46ef-a5f3-1e7c63404e1b.png)

Reply:
@omarsalem7 Do you have any tips as to why this may be happening? I'm not receiving this error on my end.